client/post-editor/index.js

```diff
@@ -14,6 +14,7 @@ module.exports = function() {
 	page( '/post', controller.pressThis, sitesController.siteSelection, sitesController.sites );
 	page( '/post/new', () => page.redirect( '/post' ) ); // redirect from beep-beep-boop
 	page( '/post/:site?/:post?', sitesController.siteSelection, sitesController.fetchJetpackSettings, controller.post );
+	page( '/post/revision/:site?/:post?', sitesController.siteSelection, sitesController.fetchJetpackSettings, controller.revision );
```

Comment:
> I think the only way adding an extra URL position in getSiteFragment would break anything is if we have a URL like /verb.with.dots/site.wordpress.com/numeric/numeric (something that would be newly and incorrectly recognized as a site ID in the length - 3 position). This seems extremely unlikely to me.

Yes it's probably a fair assessment of the risk if we add the third-to-last position only in the slug detection (which introduces asymmetry between the slug and numeric ID detections). Adding it to the numeric ID detection might be a bit riskier. (but looking at the current `page` calls I couldn't see anything ;) )

Anyway, @youknowriad is right: this isn't needed for this PR ;)